packages/client/components/ScopePhaseArea.tsx

```diff
@@ -71,33 +73,39 @@ const ScopePhaseArea = (props: Props) => {
   const {meeting} = props
   const [activeIdx, setActiveIdx] = useState(1)
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+  const {viewerMeetingMember} = meeting
+  if (!viewerMeetingMember) return null
+  const {user} = viewerMeetingMember
+  const {featureFlags} = user
+  const {gitlab: allowGitlab} = featureFlags
```

Comment:
+1 I don't know how modular the integrations should be, ideally we would check if any `cloudProvider` or `sharedProvider` is available before we show the tab, otherwise it will just be blank.

Reply:
Makes sense, I've pushed an update checking whether the `cloudProvider` & `sharedProviders` exist. I also noticed a bug in the `sharedProviders` logic and made a change.

Also, I understand that the `cloudProvider` has the GitLab Parabol app credentials which it gets from the `postDeploy` script using our env vars. But how will external users integrate their GitLab apps with Parabol? Is the `cloudProvider` currently just a placeholder that we'll update before releasing the integration?